test/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "vault_addr" {
+  description = "Vault server URL address"
+}
+
+variable "env_name" {
+  default = "test"
+}
+
+variable "envid" {
+  description = "Unique test environment identifier to prevent collisions."
+}
+
+variable "bootstrap_version" {
+  default = "master"
```

Comment:
I see this is unchanged from before though I wonder if we wanted tag. I did not expect that all master branch of infrastructure repo were relevant for deploy (as I am understand it is - see below).

----

I see:
```
fleet/templates/user_data.bash:7:bash <(curl -s https://raw.githubusercontent.com/aeternity/infra-node-aws-bootstrap/master/bootstrap.sh)
```

that contains:
```
    git -C /infrastructure fetch origin $bootstrap_version
```